dashboard/test/ui/features/foundations/i18n.feature

```diff
@@ -121,27 +117,25 @@ Scenario: Minecraft:Agent tutorial in Arabic (RTL)
   Then element ".csf-top-instructions p" has "ar-SA" text from key "data.short_instructions.MC_HOC_2017_01_RETRY"
 
 Scenario: Translated function names in Arabic
-  Given I am on "http://studio.code.org/s/allthethings/lessons/3/levels/3/lang/ar-SA"
+  Given I am on "http://studio.code.org/s/allthethings/lessons/4/levels/6/lang/ar-SA"
   And I wait for the lab page to fully load
   # Toolbox call block is translated
-  Then element "[block-id=6] .blocklyText" has "ar-SA" text from key "data.function_definitions.2-3 Artist Functions 4.draw a square.name"
+  Then element "[data-id='toolboxCallBlock'] .blocklyText" has "ar-SA" RTL text from key "data.function_definitions.2-3 Bee Functions 2.get 5.name"
```

Comment:
This is the first of 3 instances of a new "RTL text" step. This works essentially the same way, but ignores an invisible RLM character that can be found on the block text with Google Blockly.